### DISCUSSION TITLE
poster image not visible Lazy Loading

### DISCUSSION THREAD

**Discussion by rianlukog** [google/model-viewer#2537] [Q&A]
Poster image does not appear visible while waiting for model to render/load

 <style>
model-viewer{
    width: 300px;
    height: 500px;
    margin: 0 auto;
  --poster-color: transparent;

}
    </style>
</head>
<body>
    <model-viewer id="reveal" loading="eager" src="source" camera-controls auto-rotate poster="SOURCEPOSTER" ar ios-src="IOSSOURCEUSDZ"></model-viewer>
</body>
</html>

**Comment by shamsail** [comment]
Can you try with the specific `id` ? Also, Can you share how you are including `poster` and `reveal` attribute in `model-viewer` tag?
```
model-viewer#idOfYourComponent {
  --poster-color: transparent;
}
```
